packages/application-shell/src/apollo-links/header-link.ts

```diff
@@ -29,20 +30,15 @@ const isKnownGraphQlTarget = (target: GraphQlTarget) =>
 /* eslint-disable import/prefer-default-export */
 // Use a middleware to update the request headers with the correct params.
 const headerLink = new ApolloLink((operation, forward) => {
-  const { uri, cache } = operation.getContext() as ApolloContext;
-
-  // In case the `context` contains a `uri`, it means that we are not sending
-  // the request to the MC API, but to another server.
-  // For now, if that's the case, we skip the `target` validation and we do
-  // not send the custom headers.
-  if (uri) {
-    return forward(operation);
-  }
```

Comment:
This logic isn't really necessary. There is now a `skipGraphQlTargetCheck` option for that.